policyengine_us/parameters/gov/states/md/tax/income/agi/subtractions/hundred_year/amount.yaml

```diff
@@ -0,0 +1,11 @@
+description: Maryland provides this hundred year subtraction amount.
+values:
+  2021-01-01: 0
+  2022-01-01: 100_000
+metadata:
+  reference:
+    - title: MD 2022 Resident Booklet (ya)
```

Comment:
is there a legal code or bill that we can add?

Reply:
will add